src/blocks.js

```diff
@@ -52,7 +52,7 @@ export default (editor, opt = {}) => {
 
   bm.add('mj-image', {
     label: 'Image',
-    content: '<mj-image src="http://placehold.it/350x250/78c5d6/fff">',
+    content: '<mj-image src="http://placehold.it/350x250/78c5d6/fff" padding-top="0" padding-right="0" padding-left="0" padding-bottom="0"></mj-image>',
     attributes: { class: 'fa fa-image' },
     ...allBlocks,
   });
```

Comment:
Why these changes to paddings?

Reply:
The default style is incorrect if we don't add the padding zero by default.
![image](https://user-images.githubusercontent.com/46523184/71325305-c9548a80-2525-11ea-885f-67b32065af90.png)